packages/flutter/lib/src/material/dialog.dart

```diff
@@ -98,11 +105,11 @@ class Dialog extends StatelessWidget {
           child: ConstrainedBox(
             constraints: const BoxConstraints(minWidth: 280.0),
             child: Material(
-              elevation: 24.0,
-              color: _getColor(context),
-              type: MaterialType.card,
               child: child,
+              color: _getColor(context),
```

Comment:
DialogTheme should really have its own backgroundColor property and Dialog should have a backgroundColor parameter. Sadly, that means defaulting through all of them.

Reply:
Done (added to `Dialog`s and `DialogTheme)